package.json

```diff
@@ -28,11 +28,11 @@
   },
   "homepage": "https://github.com/bahmutov/npm-install#readme",
   "dependencies": {
+    "@actions/cache": "1.0.0",
```

Comment:
the latest version is 1.0.2 - should we use that?

Reply:
Seems to fix some bugs per https://github.com/actions/toolkit/blob/main/packages/cache/RELEASES.md#101 . This is normally why one would use SemVer caret `^`, but it wasn't being used here so I didn't use it.

Reply:
yeah, I prefer using exact versions of dependencies to avoid surprises

Reply:
exact pins make updates much harder for downstream consumers (including security patches) generally when it comes to libraries, but given this is wrapped up into one file with `ncc` (is that actually necessary for all GH Actions? need to read up on that), that'll happen regardless of pin or not.